ext/MathOptAIIpoptPythonCallExt.jl

```diff
@@ -38,7 +38,7 @@ function _build_set(
     hessian::Bool,
 )
     torch = PythonCall.pyimport("torch")
-    torch_model = torch.load(filename; weights_only = false)
+    torch_model = torch.load(filename; weights_only = false, map_location = "cpu")
     torch_model = torch_model.to(device)
     y = torch_model(torch.zeros(input_dimension; device))
```

Comment:
This kind of seems like it's redundant, as we send the model to the device immediately anyway.